deploy/iroha/keys/registration_service_fake@notary.priv

```diff
@@ -0,0 +1 @@
+536d9353991f0792a4bc14d2fb2b7bcbb4cd51f5a22d21ece9d1adce02d3b6a3
```

Comment:
Why we need this file?

Reply:
this key is used for the first step of a client registration:
1) we register a user account with this key in order to sign 'grant permission to BRVS' transaction created by the user
2) we actually grant needed permissions
3) the relevant user's public key is attached to the user account

Reply:
Why it's called fake?

Reply:
Just because it is used instead of relevant user key for the first step of registration. If you mind you can propose a replacement